expo/scheduler/views.py

```diff
@@ -980,12 +977,16 @@ def calendar_view(request=None,
     # I want to rewrite this to get the first day of the event from the
     # DB or conf, and then calculate the date based on the next day after
     # the start of the event.  -  HH
-    if day is not None:
-        cal_times = day_to_cal_time(day,
-                                    week=datetime(2016,
-                                                  02,
-                                                  4,
-                                                  tzinfo=pytz.timezone('UTC')))
+    conf_slug = request.GET.get('conf', None)
+    if conf_slug:
+        conf = get_conference_by_slug(conf_slug)
+    else:
+        conf = get_current_conference()
+
+    # ignore day parameter for now, day_to_cal_time ignores it
+
+    cal_times = cal_times_for_conf(conf)
```

Comment:
BLOCKER - Something is broken here - when I enter:
http://localhost:8282/scheduler/Show/Sunday?conf=GBE2016
or 
http://localhost:8282/scheduler/Show/Sunday

I still get Saturday's data.  This really did work last year, I can see it working for last year's conference here:

http://burlesque-expo.com/scheduler/Show/Sunday

We can't call this ticket fixed without this.